src/EditorFeatures/CSharpTest/Classification/SemanticClassifierTests.cs

```diff
@@ -3903,5 +3903,65 @@ public async Task NotNativeUnsignedInteger()
                 code: @"nuint i = 0;",
                 expected: Classifications(Class("nuint")));
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Classification)]
+        public async Task StaticBoldingMethodName()
+        {
+            await TestAsync(
+@"class C
+{
+    public static void Method()
+    {
+        System.Action action = Method;
+    }
+}",
+            Namespace("System"),
+            Delegate("Action"),
+            Method("Method"),
+            Static("Method"));
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Classification)]
+        public async Task StaticBoldingMethodNameNested()
+        {
+            await TestAsync(
+@"class C
+{
+    public static void Method()
+    {
+        _ = nameof(Method);
+    }
+}",
+            Keyword("_"),
+            Keyword("nameof"),
+            Static("Method"),
+            Method("Method"));
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Classification)]
+        public async Task BoldingMethodNameStaticAndNot()
+        {
+            await TestAsync(
+    @"class C
+{
+    public static void Method()
+    {
+        
+    }
+
+    public void Method(int x) 
+    {
+
+    }
+
+    public void Test() {
+        _ = nameof(Method);
+    }
+}",
+            Keyword("_"),
+            Keyword("nameof"),
+            Static("Method"),
```

Comment:
i don't know or believe that static would be correct here.  It seems like it should be bolded only if *Everything* is a static method in teh method group.